### PR TITLE
Drop support for PHP 5.x

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

**Comment by gmponos** [comment]
I don't see any problem with this.. shall we merge it?

File: src/Gravatar.php [hunk 1]
```diff
@@ -27,26 +29,16 @@
...
      * Returns an Avatar URL.
-     *
-     * @param string    $email
-     * @param array     $options
-     * @param bool|null $secure
-     *
-     * @return string
      */
-    public function avatar($email, array $options = [], $secure = null)
+    public function avatar(string $email, array $options = [], bool $secure = null): string
```
Comment (sagikazarmark) [inline src/Gravatar.php]: Secure should probably have `?bool` type.

File: src/Gravatar.php [hunk 2]
```diff
@@ -60,51 +52,32 @@ public function avatar($email, array $options = [], $secure = null)
...
     /**
      * Returns a profile URL.
-     *
-     * @param string    $email
-     * @param bool|null $secure
-     *
-     * @return string
      */
-    public function profile($email, $secure = null)
+    public function profile(string $email, bool $secure = null): string
```
Comment (sagikazarmark) [inline src/Gravatar.php]: Secure should probably have `?bool` type.

File: src/Gravatar.php [hunk 3]
```diff
@@ -60,51 +52,32 @@ public function avatar($email, array $options = [], $secure = null)
...
     /**
      * Returns a vCard URL.
-     *
-     * @param string    $email
-     * @param bool|null $secure
-     *
-     * @return string
      */
-    public function vcard($email, $secure = null)
+    public function vcard(string $email, bool $secure = null): string
```
Comment (sagikazarmark) [inline src/Gravatar.php]: Secure should probably have `?bool` type.

File: src/Gravatar.php [hunk 4]
```diff
@@ -60,51 +52,32 @@ public function avatar($email, array $options = [], $secure = null)
...
     /**
      * Returns a QR Code URL.
-     *
-     * @param string    $email
-     * @param bool|null $secure
-     *
-     * @return string
      */
-    public function qrCode($email, $secure = null)
+    public function qrCode(string $email, bool $secure = null): string
```
Comment (sagikazarmark) [inline src/Gravatar.php]: Secure should probably have `?bool` type.

**Review by sagikazarmark — COMMENTED** [review]
A few minor things I noticed

**Review by gmponos — APPROVED** [review]

**Pull request merged**